scylla/scheduler.py

```diff
@@ -111,6 +111,7 @@ def feed_from_db():
 class Scheduler(object):
 
     def __init__(self):
+        self.manager = Manager()
```

Comment:
Could you explain what's the purpose of this change?

Reply:
A legacy change. When I face `UnpickleableError: Cannot pickle <type 'thread.lock'> objects`, I thought it is due to `multiprocessing.Queue` object can not be picked.

But I finally found the problem is caused by initialize provider in the main process.

It could be removed.

Reply:
Thanks for the explanation!